patches/net/minecraft/server/packs/repository/PackRepository.java.patch

```diff
@@ -20,17 +20,17 @@
          for(RepositorySource repositorysource : this.sources) {
 -            repositorysource.loadPacks(p_143903_ -> map.put(p_143903_.getId(), p_143903_));
 +            Map<String, Pack> sourceMap = Maps.newTreeMap();
-+            repositorysource.loadPacks(p_143903_ -> sourceMap.put(p_143903_.getId(), p_143903_));
++            repositorysource.loadPacks(p_143903_ -> p_143903_.streamSelfAndChildren().forEach(p -> sourceMap.put(p.getId(), p)));
 +            map.putAll(sourceMap);
          }
  
          return ImmutableMap.copyOf(map);
 @@ -71,11 +_,12 @@
      }
  
-     private List<Pack> rebuildSelected(Collection<String> p_10518_) {
+     public List<Pack> rebuildSelected(Collection<String> p_10518_) {
```

Comment:
This shouldn't be patched public. There's a process you have to follow to add ATs in the middle of adding patches (might actually involve removing the patches, applying ATs, and then regenning source patches)

Reply:
This is not patched public, it's ATed. It just shows up in the patch as context.

Reply:
This is not patched public, it's ATed. It just shows up in the patch as context.